apps/vaporgui/TrackBall.cpp

```diff
@@ -368,11 +370,26 @@ bool Trackball::ReconstructCamera(
 
 // Set the quaternion and translation from a viewer frame
 // Also happens to construct modelview matrix, but we don't use its translation
-void Trackball::setFromFrame(
+int Trackball::setFromFrame(
```

Comment:
Our convention for error handling is that functions return an integer status if they report errors  with MyBase. Otherwise they should return a boolean.

Reply:
Fixed.